Dockerfile

```diff
@@ -1,9 +1,2 @@
-FROM belly/buildstep
-
-# Run build against Gemfile first so subsequent builds are much faster
-ADD Gemfile /app/Gemfile
-ADD Gemfile.lock /app/Gemfile.lock
-RUN /build/builder
-
-ADD . /app
+FROM belly/napa
 CMD /start clock
```

Comment:
We put a simple one in, so we could move from buildstep easier - https://github.com/bellycard/napa/blob/master/contrib/start.rb